sql/hive/src/test/scala/org/apache/spark/sql/hive/orc/OrcQuerySuite.scala

```diff
@@ -348,7 +348,7 @@ class OrcQuerySuite extends QueryTest with BeforeAndAfterAll with OrcTest {
     }
   }
 
-  test("SPARK-10623 Enable ORC PPD") {
+  ignore("SPARK-10623 Enable ORC PPD") {
```

Comment:
Filed [SPARK-13728](https://issues.apache.org/jira/browse/SPARK-13728).  I'm not sure why this is broken as we are still passing down the filters correctly.  Since this feature is turned off by default it seems okay to break it temporarily.